### DISCUSSION TITLE
`browser_style` Not Showing Up in Final Manifest on Side Panel (Firefox MV2)

### DISCUSSION THREAD

**Discussion by n4ze3m** [wxt-dev/wxt#913] [Q&A] [answered]
Hey, in a Firefox Manifest V2 extension, the browser_style property is not appearing in the final manifest for the side panel. Is there a way to fix it?

**Comment by aklinker1** [comment]
How or where are you setting the value to `true`?

**Reply by n4ze3m** [reply]
Hey, I need to set browse_style to false since browse_style: true is the default, and I guess it's causing issues with text selection.
 I added it like this in the docs. open_at_install is working fine, but browse_style is not showing up in the manifest file. 

 ![image](https://github.com/user-attachments/assets/6ece3479-3db0-4e80-b8eb-d39f6a407179)

 Anyway, I really like your framework—it's really easy to use and provides a great dx.

code: 
https://github.com/n4ze3m/page-assist/blob/066db5dfb747b30c2e1594cca0848255a47a5cf1/src/entries/sidepanel/index.html#L8

**Reply by aklinker1** [reply, marked as answer]
Looks like I have a typo causing WXT to look for `browse_style` instead of `browser_style` 🤦 

https://github.com/wxt-dev/wxt/blob/2fd3503694f1b668223ed5bc6756b89aeef26649/packages/wxt/src/core/utils/building/find-entrypoints.ts#L390

https://github.com/wxt-dev/wxt/pull/914

**Reply by aklinker1** [reply]
Released fix in [`v0.19.6`](https://github.com/wxt-dev/wxt/releases/tag/wxt-v0.19.6)
